### PR TITLE
Updates for latest Python support

### PR DESCRIPTION
**Pull request by djcomlab**
Changes as per issue reported in https://github.com/galaxyproject/galaxy/pull/11452#issuecomment-784761019

Somebody (@proccaserra @Zigur ?) will need to tag, release and make available in PyPI so that Galaxy can keep working with the Galaxy ISA datatype.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -22,12 +22,10 @@
         'Intended Audience :: Developers',
         'Intended Audience :: System Administrators',
         'Topic :: Scientific/Engineering :: Bio-Informatics',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3.6'
```
Comment (nsoranzo) [inline setup.py]:
```suggestion
        'Programming Language :: Python :: 3',
        'Programming Language :: Python :: 3.6',
        'Programming Language :: Python :: 3.7',
        'Programming Language :: Python :: 3.8',
        'Programming Language :: Python :: 3.9',
```

**Comment by nsoranzo** [comment]
Thanks a lot @djcomlab !

**Review by Zigur — APPROVED** [review]

**Comment by nsoranzo** [comment]
Thanks @Zigur for the merge and the GitHub release! Unfortunately it seems it hasn't been uploaded to PyPI yet, the GitHub workflow did not run (looks like https://github.community/t/workflow-set-for-on-release-not-triggering-not-showing-up/16286 ).
Can you please upload it manually?

**Comment by Zigur** [comment]
@nsoranzo, thanks for the hint. Will upload it now.

**Comment by Zigur** [comment]
This is uploaded now: https://pypi.org/project/isa-rwval/0.10.10/

**Comment by nsoranzo** [comment]
Great, thanks @Zigur !

**Pull request merged**